framework/resources/Functional/values/saffron_encoding_DRILL_5772.q

```diff
@@ -0,0 +1,5 @@
+drop table if exists encoding_test;
+create table encoding_test as select name, alias, description from (values('Василий Теркин', 'Вася', 'персонаж'), ('王維', '摩詰', '詩人'), ('Ибн Рушд', 'ابن رشد', 'философ'), ('James Cook', 'James', 'navigator')) as t(name, alias, description);
+select name from encoding_test where alias in ('Вася', '摩詰', 'ابن رشد');
+drop table if exists encoding_test;
+select 1 from (values(0));
```

Comment:
@omalashevsky You can mark the query that you want to verify result with "--@test". Doing it this way, you don't need the last query. I believe the README file explains it.  After correction, and rerun pre-commit, if it passes, you can merge the PR. +1. Thanks.